simplq/pom.xml

```diff
@@ -1,135 +1,162 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>me.simplq</groupId>
-    <artifactId>simplq-backend</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
-    <relativePath>../pom.xml</relativePath>
-  </parent>
-  <artifactId>simplq</artifactId>
-  <version>0.0.1-SNAPSHOT</version>
-  <name>simplq-server</name>
-  <packaging>jar</packaging>
-  <description>SimplQ Backend Server</description>
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>me.simplq</groupId>
+		<artifactId>simplq-backend</artifactId>
+		<version>0.0.1-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+	<artifactId>simplq</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>simplq-server</name>
+	<packaging>jar</packaging>
+	<description>SimplQ Backend Server</description>
 
-  <properties>
-      <junitVersion>5.7.0</junitVersion>
-      <mockitoVersion>3.7.7</mockitoVersion>
-  </properties>
+	<properties>
+		<junitVersion>5.7.0</junitVersion>
+		<mockitoVersion>3.7.7</mockitoVersion>
+		<db.username>dev</db.username>
+		<db.password>password</db.password>
+		<db.url>jdbc:postgresql://localhost:5432/simplq</db.url>
```

Comment:
One final ask. Lets remove these defaults, they don't make sense.

I asked to give defaults thinking they will be used for the local in memory DB. But I was confused, sorry.

Reply:
Well, I will live them. Because if I want to run some postresql locally to do some tests on migrations, I will not have to  write every time `-D db.username=...`